src/server/trpc/context.ts

```diff
@@ -11,16 +12,17 @@ type CreateContextOptions = Record<string, never>;
  * - trpc's `createSSGHelpers` where we don't have req/res
  * @see https://create.t3.gg/en/usage/trpc#-servertrpccontextts
  **/
-export const createContextInner = async (opts: CreateContextOptions) => {
-  return {};
-};
+// export const createContextInner = async (opts: CreateContextOptions) => {
```

Comment:
I couldn't make sense of this. we can rework this in a future PR. But I added the session below to pass to the calls. This may or not be the correct way to do this?

Reply:
I think, this is a function to create the context inner value regarding given Options. As a default, it creates a Context data object with an empty object {}.
You put the inner values of the context directly into the createContext function, what looks easier, but may be not flexible enough for complex context objects (like shopping bags within an ecommerce app). 
Should be ok, as long as context only carries the single session object.